.changeset/hungry-dingos-move.md

```diff
@@ -0,0 +1,9 @@
+---
+'@directus/api': minor
+'@directus/validation': patch
+'@directus/env': patch
+'@directus/app': patch
+'@directus/utils': patch
```

Comment:
This seems to be in error or leftover from a previous change. No changes to utils has been made

```suggestion
```